indra/llappearanceutility/llbakingavatar.h

```diff
@@ -66,43 +66,41 @@ class LLBakingAvatar : public LLAvatarAppearance
     // LLCharacter interface
     //--------------------------------------------------------------------
 public:
-    /*virtual*/ LLVector3       getCharacterPosition();
-    /*virtual*/ LLQuaternion    getCharacterRotation();
-    /*virtual*/ LLVector3       getCharacterVelocity();
-    /*virtual*/ LLVector3       getCharacterAngularVelocity();
-
-    /*virtual*/ const LLUUID&   getID() const;
-    /*virtual*/ void            addDebugText(const std::string& text);
-    /*virtual*/ F32             getTimeDilation();
-    /*virtual*/ void            getGround(const LLVector3 &inPos, LLVector3 &outPos, LLVector3 &outNorm);
-    /*virtual*/ F32             getPixelArea() const;
-    /*virtual*/ LLVector3d      getPosGlobalFromAgent(const LLVector3 &position);
-    /*virtual*/ LLVector3       getPosAgentFromGlobal(const LLVector3d &position);
+    LLVector3       getCharacterPosition() override;
+    LLQuaternion    getCharacterRotation() override;
+    LLVector3       getCharacterVelocity() override;
+    LLVector3       getCharacterAngularVelocity() override;
+
+    const LLUUID&   getID() const override;
+    void            addDebugText(const std::string& text) override;
+    F32             getTimeDilation() override;
+    void            getGround(const LLVector3 &inPos, LLVector3 &outPos, LLVector3 &outNorm) override;
+    F32             getPixelArea() const override;
+    LLVector3d      getPosGlobalFromAgent(const LLVector3 &position) override;
+    LLVector3       getPosAgentFromGlobal(const LLVector3d &position) override;
 
     //--------------------------------------------------------------------
     // LLAvatarAppearance interface
     //--------------------------------------------------------------------
 public:
-    /*virtual*/ void    bodySizeChanged();
```

Comment:
In this block of `/*virtual*/` -> `override` changes, it's odd to see a couple method declarations vanish: `bodySizeChanged()` and `isUsingServerBakes()`. I'm not objecting, but I'd like a word of explanation in this PR.